### ISSUE TITLE
No results are showed

### ISSUE THREAD

**Issue by pinkybados** [kamshory/ZKLibrary#32]
There is no results in when i run the test.php file. There is another thing that I need to do?

**Comment by tekrajchhetri** [comment]
Hi @kamshory, I am also getting the same problem. The device is ZKTeco K40 with firmware version 4.0.1

**Comment by pinkybados** [comment]
Thanks @kamshory. Now I am descarting all the possibilities, maybe the php vesion is different, I am testing on XAMPP v.3.1.0 and PHP v5.4.7. Is the library compatible? 
Also The firmware device is 4.0.1(build 74), Is the firmware compatible?

**Comment by bjrukin** [comment]
I don't think it is php version that is causing the issue. k40 is not supported.

**Comment by DJSamet** [comment]
I am new here, i am using ZKTime eco K40. 
I get no response except for "Cheers". 
Below is my code. . . . Please Help Me, I am using XAMPP.

Yes, The datas do exists.
Yes, I've Enabled sockets.dll and soap.dll in php.ini file.

I am very amateur... Give me suggestions.. What is the problem?



<body>
		<h1>Cheers</h1> 
		<?php
		include "zklibrary.php";
		$zk = new ZKLibrary('192.168.100.201', 4370);
		$zk->connect();
		$zk->disableDevice();
		$users = $zk->getUser();
		foreach($users as $key=>$user)
		{
			echo $user[0]; //NO RESULT
			echo $user[1]; //NO RESULT
			echo $user[2]; //NO RESULT
			echo("OK"); //NO RESULT
		}
		$zk->enableDevice();
		?>
	</body>

**Comment by DJSamet** [comment]
@tekrajchhetri : Bro, kei pragati xa ta ZKTime eco K-40 device ma??? Tapaile kei milaunuvo vane malai ni vannu haai....

**Comment by tekrajchhetri** [comment]
@DJSamet : After some research, I got to know that it needs to have push data technology in order to be able to send data back, which this device doesn't have. Using this device it's not possible for what I wanted to achieve i.e. Push data to cloud / remote server and access via the internet. If you are working on the same thing then I guess we're working on the wrong device.

**Comment by DJSamet** [comment]
@tekrajchhetri : Bro, teso vaye kun chai device use garda easy, cheap ra best hunxa?? Any suggestions??

**Comment by tekrajchhetri** [comment]
@DJSamet : check this out [GBS(WiFi) Biometric Attendance Management with Unlimited Cloud Storage for Lifetime at zero maintenance, Android and iOS App support, Reports Downloadable Globally](https://www.amazon.in/dp/B06X92F7GF/ref=sspa_dk_detail_0?psc=1&pd_rd_i=B06X92F7GF&pf_rd_m=A1VBAL9TL5WCBF&pf_rd_p=4712422188187354270&pf_rd_r=YFSYNJRMTG5YNX62JQQH&pd_rd_wg=V1vpn&pf_rd_s=desktop-dp-sims&pf_rd_t=40701&pd_rd_w=E6D9z&pf_rd_i=desktop-dp-sims&pd_rd_r=7b66e5c0-72dd-11e8-824c-ed4f524f6e0b) . You literally have to do nothing, they'll provide lifetime hosting for free, also they have Apps for both IOS and Android. You wouldn't get any products in such price.

**Comment by DJSamet** [comment]
Thank You For Your Suggestion!!! The device you suggest is very very nice. . . . But, the problem is that i need to build it on pure nepali (nepali_website) . . . So, i have a question @tekrajchhetri . . . Does that GBS device's cloud server provide us the capability to fetch datas and upload user datas from their server to ours ? ? ?
